data/ABRS_2023/metadata.yml

```diff
@@ -32,13 +32,71 @@ dataset:
+        entity_measured = ifelse(category %in% c("tertiary_pinnae"), "tertiary_pinnae", entity_measured),
+        entity_measured = ifelse(category %in% c("quaternary_pinnae"), "quaternary_pinnae", entity_measured),
+        entity_measured = ifelse(category %in% c("largest_pinnae", "pinnae", "pinnules", "pinnae or pinnule", "pinnule", "longest_pinnae","lateral_pinnae", "terminal_pinnae"), "pinnae", entity_measured),
+        entity_measured = ifelse(category %in% c("proximal_pinnae"), "proximal_pinnule", entity_measured),
+        entity_measured = ifelse(category %in% c("distal_pinnae"), "distal_pinnule", entity_measured),
+        entity_measured = ifelse(category %in% c("upper_pinnae"), "upper_pinnae", entity_measured), 
+        entity_measured = ifelse(category %in% c("apical_pinnules"), "apical_pinnule", entity_measured), 
+        entity_measured = ifelse(category %in% c("lower_pinnae", "basal_pinnae", "lowest_pinnae"), "basal_pinnae", entity_measured),         
+        entity_measured = ifelse(category %in% c("fertile_pinnae", "fertile_pinnules"), "fertile_pinnule", entity_measured),
+        entity_measured = ifelse(category %in% c("sterile_pinnules", "sterile_pinnae"), "sterile_pinnule", entity_measured),
+        entity_measured = ifelse(category %in% c("scale_leaf", "scale_leaves", "cataphylllamina", "involucral_lamina"), "scale_leaf", entity_measured),
+        entity_measured = ifelse(category %in% c("lower_leaf", "lower_leaves", "lower_leaf_lamina", "lower_lamina", "lowest_leaves"), "lower_leaf", entity_measured),
+        entity_measured = ifelse(category %in% c("upper_leaf", "upper_leaves", "upper_leaf_lamina", "laminas", "upper_lamina"), "upper_leaf", entity_measured),
+        entity_measured = ifelse(category %in% c("stemleaves", "lateral_stem_leaves", "main_stem_leaves","stem_leaves","flowerin_stem_leaves","vegetative_stem_leaves", "erect_stem_leaves", "stem_lamina", "fruiting_stem_leaves", "lateral_stemleaf_lamina"), "stem_leaf", entity_measured),
+        entity_measured = ifelse(category %in% c("lateral_leaf", "lateral_leaves", "axillary_shoot_leaves", "axillary_stem_leaves"), "lateral_leaf", entity_measured),
+        entity_measured = ifelse(category %in% c("cauline_leaves", "cauline_lamina", "caulineleaves", "cauline_leaf"), "cauline_leaf", entity_measured),
+        entity_measured = ifelse(category %in% c("basal_leaves", "radical_leaves", "basal_eaves", "nest_fronds", "radical_leaf", "basal_lamina","basal_leaf", "rosette_leaves"), "basal_leaf", entity_measured),
+        entity_measured = ifelse(category %in% c("phyllode", "phyllodes"), "phyllode", entity_measured),
+        entity_measured = ifelse(category %in% c("cladode", "cladodes", "phylloclades"), "cladode", entity_measured),
+        entity_measured = ifelse(category %in% c("leafsegments"), "lobes", entity_measured),
+        entity_measured = ifelse(category %in% c("lobes", "stem_leaf", "leaf"), "all", entity_measured),
```

Comment:
Does this line mean that lobes, stem_leaf or leaf categories will be replaced by "all"?

Reply:
> Does this line mean that lobes, stem_leaf or leaf categories will be replaced by "all"?

Yes, that was in one of your scripts, but maybe I was meant to change it? Feel free to adjust anything.